internal/templates/render/render.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package render
```

Comment:
Package comment.

Or is "internal" really necessary?

Reply:
Added package comment, will move also.

Reply:
Could you elaborate on why you'd like to not use `internal`?

Reply:
It's just that we haven't been using `internal` other than version. My understanding is not deep enough to tell when or not to use `internal`. Thus the question.

Reply:
I stopped using "internal" for consistency with our other repos.